shop/shop/src/main/java/ro/msg/learning/shop/mapper/ProductMapper.java

```diff
@@ -22,7 +23,7 @@ public ProductDto toDto(Product product) {
         String imageUrl = product.getImageUrl();
         ProductCategoryDto categoryDto = new ProductCategoryDto(categoryId, product.getCategory().getName(), product.getCategory().getDescription());
         SupplierDto supplierDto = new SupplierDto(supplierId, product.getSupplier().getName());
-        return new ProductDto(productId, categoryId, supplierId, name, description, price, weight, categoryDto, supplierDto, imageUrl);
```

Comment:
Did you reverted by mistake an old commit? I think you made the switch from `id` to `productId` and now it's reversed

Reply:
Am lucrat si la urmatoarele capitole pe acelasi branch ca si la ultimul commit si dupa cand am incercat sa ma mut pe alt branch am avut niste probleme, de asta in unele locuri s-ar putea sa apara varianta gresita, scuzee